pom.xml

```diff
@@ -51,6 +51,7 @@
         <primefaces.version>7.0</primefaces.version>
         <thymeleaf.version>3.0.10.RELEASE</thymeleaf.version>
         <junit.version>4.12</junit.version>
+        <resteasy.version>4.3.0.Final</resteasy.version>
```

Comment:
No CQ (approval) for RestEasy 4.3.0.

Reply:
@cealsair  Could you help me on this one, please?

Reply:
No need to do any more CQ. Check against https://clearlydefined.io/. See the latest [Third Party Dependency wiki ](https://wiki.eclipse.org/MicroProfile/3rdPartyDependencyProcess)I updated.